tools/ARIAtools/ARIAProduct.py

```diff
@@ -106,29 +117,23 @@ def __readproduct__(self, fname):
         '''
 
         ### Get standard product version from file
-        # If netcdf with groups
-        if fname.startswith('http'):
-            fname_orig = fname
-            gdal.SetConfigOption('GDAL_HTTP_COOKIEFILE','cookies.txt')
-
-            fname   = '/vsicurl/{}'.format(fname)
-            version ='URL'
-            fmt     = 'HDF5:"'
-        else:
-            try:
+        try:
+            #version accessed differently between URL vs downloaded product
+            if 'https://' in fname:
+                version=str(gdal.Open(fname).GetMetadata_Dict()['version'])
+            else:
```

Comment:
Reading of 'NC_GLOBAL#version' should not be different if in S3.
To be tested on opensarlabs as it needs the netcdf to be recognized!

Reply:
Yup, will have to confirm on opensarlabs, once the technical issue on my end gets resolved. From my end locally though, I could successfully access the version from the URLs this way.